clarsach/spectrum.py

```diff
@@ -30,17 +30,17 @@ def __init__(self, filename, telescope='HETG'):
         if self.bin_unit != self.rmf.energ_unit:
             print("Warning: RMF units and pha file units are not the same!!!")
 
-
-        # Might need to notice or group some day
-        #self.notice = np.ones(len(self.counts), dtype=bool)
-        #self.group  = np.zeros(len(self.counts), dtype=int)
-
-
     def __store_path(self, filename):
         self.path = '/'.join(filename.split('/')[0:-1]) + "/"
-
         return
 
+    def apply_resp(self, mflux):
+        # Given a model flux spectrum, apply the response
+        mrate  = self.arf.apply_arf(mflux)  # phot/s per bin?
+        result = self.rmf.apply_rmf(mrate)  # count/s per bin?
+        result *= self.exposure * self.arf.fracexpo  # counts per bin
```

Comment:
Do you think using `exposure` and `fracexpo` should be the default behaviour for *all* instruments? If so, it might be worth shoving it into the respective `RMF` or `ARF` method. 

And I'm not sure, but I thought `exposure` and `fracexpo` were applied before the RMF? I don't have a good reasoning for why I think that, apart from that I think I've read it somewhere.